stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -578,7 +578,8 @@ extension ProcessTerminationStatus {
       return CInt(signal) == SIGILL || CInt(signal) == SIGTRAP
     default:
       // This default case is needed for standard library builds where
-      // resilience is enabled
+      // resilience is enabled.
+      // FIXME: Need a way to suppress when not.
       return false
```

Comment:
@slavapestov is there any technique we can use to suppress the warning for the default when resilience _isn't_ enabled? Like with a #if for when resilience is turned on?